_posts/0000-01-02-salokajalan.md

```diff
@@ -0,0 +1 @@
+Welcome to our second slide
```

Comment:
a='''Twinkle, twinkle, little star,\n
    How I wonder what you are! \n
        Up above the world so high,\n           
        Like a diamond in the sky.\n 
Twinkle, twinkle, little star, \n
    How I wonder what you are.'''
print(a)